apps/explorer/src/components/TableTitleButton/TableTitleButton.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="table-title-button"></div>
+</template>
+
+<script>
+export default {
```

Comment:
Let's try to use [vue-class-componnent](https://github.com/vuejs/vue-class-component) syntax to unify with other components

Reply:
I'll take a look at it.